engine/src/main/java/org/hibernate/search/bridge/KeepSafeFromAnalyzerBridge.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.bridge;
+
+
+/**
+ * This interface is used to mark the bridges we want to keep safe from the analyzers.
+ *
+ * @author Guillaume Smet
+ */
+public interface KeepSafeFromAnalyzerBridge {
```

Comment:
if I understand correctly this interface is used in DSL when we shouldn't apply an analyzer (basically is the same as calling `.ignoreAnalyzer()`). Am I right?

Shouldn't this be in the package: `org.hibernate.search.query.dsl[.impl]`